pkg/assets/assets.go

```diff
@@ -1,20 +1,18 @@
 package assets
 
 import (
-	"bytes"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"text/template"
+	"strings"
 )
 
 type Permission os.FileMode
 
 const (
 	PermissionDirectoryDefault Permission = 0755
 	PermissionFileDefault      Permission = 0644
-	PermissionFileRestricted   Permission = 0600
```

Comment:
I think this is important to make it really easy for people to indicate their secrets.  We don't want to render 0644 if we can help it.

Reply:
readded